src/themes/stories/TokenShowcase.tsx

```diff
@@ -1,27 +1,32 @@
-import React, { FC } from 'react';
-import { SystemConfig, Table, useChakraContext } from '@chakra-ui/react';
+import React from 'react';
+import {
+  SystemConfig,
+  Table,
+  Tokens,
+  useChakraContext,
+} from '@chakra-ui/react';
 
 type TokenGroupName = keyof NonNullable<
   NonNullable<SystemConfig['theme']>['tokens']
 >;
 
-type TokenShowcaseProps = {
+type TokenShowcaseProps<Token extends keyof Tokens> = {
   tokenPath: string;
   renderValue?: boolean;
-  renderDemo?: (tokenValue: string) => React.ReactNode;
+  renderDemo?: (tokenName: Tokens[Token]) => React.ReactNode;
```

Comment:
If we pass `string` and want to use it as a token value, strict tokens would yell at us.
This is why we add type parameters so we can properly cast a token name to an expected token.